core-ui/src/components/Predefined/Create/ServiceBindings/SecretRefForm/SecretRefForm.js

```diff
@@ -86,128 +46,106 @@ export function SecretRefForm({
+    }
+    const secret = secrets.find(s => s.metadata.name === ref.name);
+    return secret?.data ? tryBase64Decode(secret.data[ref.key]) : '';
   };
 
-  const getRefValidation = (ref, otherRefs) => {
+  const getRefValidationMessage = ref => {
     try {
-      if (!ref.secretKeyRef.name) {
-        return t('btp-service-bindings.create.ref-form.choose-secret');
-      } else if (!ref.secretKeyRef.key) {
-        return t('btp-service-bindings.create.ref-form.choose-key');
-      } else if (!isValidJSONObject(getSecretValue(ref))) {
-        return t(
-          'btp-service-bindings.create.ref-form.value-must-be-json-object',
-        );
+      if (ref?.key && !isValidJSONObject(getSecretValue(ref))) {
+        return t('btp-service-bindings.messages.value-must-be-json-object');
       }
 
-      const duplicates = otherRefs.filter(
-        r =>
-          r.secretKeyRef.name === ref.secretKeyRef.name &&
-          r.secretKeyRef.key === ref.secretKeyRef.key,
-      );
+      const duplicates = secretRefs.filter(r => {
+        console.log('filter', r, ref);
+        return (
+          r.secretKeyRef.name === ref.name && r.secretKeyRef.key === ref.key
+        );
+      });
+      console.log('duplicates', duplicates);
 
       if (duplicates.length > 1) {
-        return t('btp-service-bindings.create.ref-form.duplicate-ref');
+        return t('btp-service-bindings.messages.duplicate-ref');
       }
     } catch (e) {
       console.warn(e);
-      return t('btp-service-bindings.create.ref-form.invalid-ref');
+      return t('btp-service-bindings.messages.invalid-ref');
     }
     return '';
   };
-
-  const addRefButton = () => {
-    const text = t('common.buttons.add-new');
-    if (loading || error) {
-      return (
-        <Tooltip
-          content={loading ? t('common.headers.loading') : error.message}
-        >
-          <Button glyph="add" disabled>
-            {text}
-          </Button>
-        </Tooltip>
-      );
-    }
-    return (
-      <Button glyph="add" onClick={addRef}>
-        {text}
-      </Button>
-    );
-  };
-
-  const onSecretSelect = ref => (_, selected) => {
-    ref.secretKeyRef.name = selected.text;
-    ref.secretKeyRef.key = '';
-    setRefs([...refs]);
-  };
-
-  const onKeySelect = ref => (_, selected) => {
-    ref.secretKeyRef.key = selected.text;
-    setRefs([...refs]);
+  const getRefValidation = ref => {
+    const message = getRefValidationMessage(ref);
+    return message
+      ? {
+          state: 'error',
+          text: message,
+        }
+      : undefined;
   };
 
-  const renderRefs = refs
-    .filter(ref => ref.secretKeyRef)
-    .map(ref => ({
-      ...ref,
-      renderKey: uuid(),
-      validationMessage: getRefValidation(ref, refs),
-    }));
-
   return (
-    <>
-      <ul className="secret-ref-form">
-        {renderRefs.map((ref, index) => (
-          <li key={ref.renderKey}>
-            <RefValidationMessage message={ref.validationMessage} />
+    <MultiInput
```

Comment:
Please fix the placeholder
<img width="644" alt="Screen Shot 2021-10-20 at 14 31 03" src="https://user-images.githubusercontent.com/40663133/138092959-d5537da6-03fe-49c5-af85-9152253dc67f.png">